core/src/main/java/org/apache/accumulo/core/iterators/ColumnFamilySkippingIterator.java

```diff
@@ -16,7 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.accumulo.core.iteratorsImpl.system;
+package org.apache.accumulo.core.iterators;
```

Comment:
Moving the interface back to public API seems like the obvious fix, but are we expecting users to actually use this interface? I haven't followed all the details, but my understanding was that this was a feature intended solely for internal use with some of our own system iterators. But, maybe I'm wrong? Is this something that is useful to user iterators and safe for them to use?

Reply:
That is what I had originally thought but after investigating this bug it seems like `SortedMapIterator` and this one could be cast to an InterruptibleIterator in the `LocalityGroupIterator`
https://github.com/apache/accumulo/blob/8d33bc4d489880f15950189d20287262ce46e68e/core/src/main/java/org/apache/accumulo/core/iteratorsImpl/system/LocalityGroupIterator.java#L49

I think if a user extends one of them and configures it to run in certain situations it could be created through deepCopy(). But it also seems dangerous either way.

Reply:
Also, my original intention in #1411 was since SortedMapIterator was used so much in our Tests that it would be useful to users as well. But I could be wrong in that thinking. It could be unsafe to expose it to users so a better option may be to move it in with the other system iterators. I can't remember if I attempted this or how difficult this would be though.

Reply:
Moved this iterator back to iteratorsImpl package in 1a4bb21607324ef6a9d2a0eeb458c0cdb3c59681